docs/api.md

```diff
@@ -226,6 +226,9 @@ Indicates that the browser is connected.
   - `videoSize` <[Object]> Specifies dimensions of the automatically recorded video. Can only be used if `videosPath` is set. If not specified the size will be equal to `viewport`. If `viewport` is not configured explicitly the video size defaults to 1280x720. Actual picture of the page will be scaled down if necessary to fit specified size.
     - `width` <[number]> Video frame width.
     - `height` <[number]> Video frame height.
+  - `har` <[Object]> Enables [HAR](http://www.softwareishard.com/blog/har-12-spec) recording for all the pages into `har.path` file. If not specified, HAR is not recorded. Make sure to await [`browserContext.close`](#browsercontextclose) for HAR to be saved.
```

Comment:
Im thinking about naming this `captureHAR` or `recordHAR` or something. `har` by itself makes me think of passing a har file into this context. And then it would serve all network requests from my har.